### PR TITLE
Backup readme txt changes

### PR DESCRIPTION
**Pull request by mikestottuk**
## Proposed changes:

Not sure this actually needs a PR - but anyway. Some small changes to the plugin author name, the readme.txt 

## Jetpack product discussion

* ?

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Check for typos

### CODE REVIEW

**Comment by singerb** [comment]
Note - post-review, we can deploy the readme changes to SVN right away.

File: projects/plugins/backup/jetpack-backup.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * Plugin URI: https://jetpack.com/jetpack-backup
  * Description: Easily restore or download a backup of your site from a specific moment in time.
  * Version: 1.8-alpha
- * Author: Automattic
+ * Author: Automattic - Backup team
```
Comment (jeherve) [inline projects/plugins/backup/jetpack-backup.php]: I think we've used "Jetpack" in there for the other plugins:
https://wordpress.org/plugins/jetpack-protect/
https://wordpress.org/plugins/jetpack-boost/

```suggestion
 * Author: Automattic - Jetpack Backup team
```

Or maybe

```suggestion
 * Author: Automattic - Jetpack Security team
```

Comment (singerb) [reply projects/plugins/backup/jetpack-backup.php]: We do, yes; good catch. I think `Jetpack Backup team` is better for what we're going for here; we use Security for the Protect plugin.

File: projects/plugins/backup/readme.txt [hunk 2]
```diff
@@ -14,6 +14,8 @@ VaultPress is the most proven WordPress backup plugin with over 270 million back
 
 Jetpack VaultPress Backup is a plugin that gives you cloud-based backups and one-click restores without needing to install the full Jetpack plugin.
 
+VaultPress is the most proven WordPress backup plugin with over 270 million backups. Save each change and get back online fast with one-click restores.
```
Comment (singerb) [inline projects/plugins/backup/readme.txt]: Noting there was discussion in Slack about this line as well as the line below it was copied from. p1682524957608919/1682100655.060309-slack-C01264051NE

**Comment by mikestottuk** [comment]
I've made two small tweaks as discussed above. 

* Automattic - Jetpack Backup 

As well as adding Jetpack VaultPress Backup to the short description. I also re-ordered it a little so that one-click restores is first up before most proven solution (which we repeat anyway in the description now). 

Screenshot_20230426-205311

**Review by pypt — APPROVED** [review]

**Comment by pypt** [comment]
Done:

Screenshot 2023-05-03 at 16 56 32

Screenshot 2023-05-03 at 16 56 45

**Pull request merged**